Assets/MixedRealityToolkit/_Core/Extensions/InputActionsExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Extensions
+{
+    /// <summary>
+    /// Extension methods for the Mixed Reality Toolkit InputActions arrray
+    /// </summary>
+    public static class InputActionsExtensions
+    {
+        /// <summary>
+        /// Get an Input Action by name from the current Array
+        /// </summary>
+        /// <param name="input">InputAction Array to query</param>
+        /// <param name="actionName">Input Action name to search for</param>
+        /// <returns>A specific InputAction if found, else null</returns>
+        public static InputAction GetActionByName(this InputAction[] input, string actionName)
```

Comment:
Why do we need this?

Reply:
It was added orriginally when we had an InputActions array, it is a little redundant now. However, I'd like to keep it for now as we haven't yet fully settled and re can remove this in the Alpha pass

Reply:
Yeah I'd like to discourage people trying to look for actions by string